extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcTracingRuntimeConfig.java

```diff
@@ -15,13 +15,15 @@ public class DataSourceJdbcTracingRuntimeConfig {
     public Optional<Boolean> enabled = Optional.empty();
 
     /**
-     * Trace calls with active Spans only
+     * Trace calls with active Spans only.
+     * Please note that this configuration property is only supported with OpenTracing.
```

Comment:
I think the plan is to retire OpenTracing so I would rather not add this line. But @radcortez will confirm.

Reply:
@gsmet this property can't be there for OpenTelemetry as it is not used by it. If OpenTracing is going to be retired, we can remove the property. There need to be a way to tell users to ignore it for OTEL. @radcortez WDYT?

Reply:
OpenTracing will not be retired immediately. We haven't decided on sunset process.